test/distributed/org/apache/cassandra/distributed/test/NetstatsRepairStreamingTest.java

```diff
@@ -76,7 +76,7 @@ private void executeTest(boolean compressionEnabled) throws Exception
 
             final Future<NetstatResults> resultsFuture1 = executorService.submit(new NetstatsCallable(node1));
 
-            node1.nodetoolResult("repair", "netstats_test").asserts().success();
+            node1.nodetoolResult("repair", "--full", "netstats_test").asserts().success();
```

Comment:
Is this related to the patch?

Reply:
Nope - slipped through from another diff. I'll fix.